website/content/docs/k8s/deployment-configurations/external-service.mdx

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+page_title: Register services running on external nodes to Consul on Kubernetes
+description: >-
+  Learn how to register a service running on an external node to the Consul catalog when running Consul on Kubernetes.
+---
+
+# Register services running on external nodes to Consul on Kubernetes
+
+This page provides an overview for registering services in the Consul catalog when the service runs on a node that is not part of the Kubernetes cluster.
+
+## Introduction
+
+Because Kubernetes has built-in service discovery capabilities, Consul on Kubernetes includes components such as [Consul dataplanes](/consul/docs/connect/dataplane) and [service sync](/consul/docs/k8s/service-sync) so that operators can continue to use Kubernetes tools and processes. However, this approach to service networking still requires a service to run on a node that Consul is aware of, either through a local workload agent or the Kubernetes cluster. We call services that run on external nodes that Consul cannot automatically recognize _external services_.
```

Comment:
```suggestion
Because Kubernetes has built-in service discovery capabilities, Consul on Kubernetes includes components such as [Consul dataplanes](/consul/docs/connect/dataplane) and [service sync](/consul/docs/k8s/service-sync) so that operators can continue to use Kubernetes tools and processes. However, this approach to service networking still requires a service to run on a node that Consul is aware of, either through a local Consul client agent or the Kubernetes cluster. We call services that run on external nodes that Consul cannot automatically recognize _external services_.
```